address;source_location;insn;indent
0x511cb0;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1749;PUSH	%RBP;
0x511cb1;:0;SUB	$0x20,%RSP;
0x511cb5;/usr/local/src/conda/python-3.12.0/Include/object.h:967;MOV	%FS:0x28,%RAX;
0x511cbe;:0;MOV	%RAX,0x18(%RSP);
0x511cc3;:0;MOV	0x8(%RDI),%RAX;
0x511cc7;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1755;TESTB	$0x20,0xab(%RAX);
0x511cce;:0;JE	4275fd <PyDict_GetItemWithError.cold+0x7>;
0x511cd4;:0;MOV	%RDI,%RBP;
0x511cd7;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1759;CMPQ	$0x93b8c0,0x8(%RSI);
0x511cdf;:0;JNE	511d10 <PyDict_GetItemWithError+0x60>;
0x511ce1;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:284;MOV	0x18(%RSI),%RDX;
0x511ce5;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1759;CMP	$-0x1,%RDX;
0x511ce9;:0;JE	511d10 <PyDict_GetItemWithError+0x60>;
0x511ceb;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1767;LEA	0x10(%RSP),%RCX;
0x511cf0;:0;CALL	505980 <_Py_dict_lookup>;
0x511cf5;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1769;MOV	0x10(%RSP),%RAX;
0x511cfa;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1770;MOV	0x18(%RSP),%RDX;
0x511cff;:0;SUB	%FS:0x28,%RDX;
0x511d08;:0;JNE	511d43 <PyDict_GetItemWithError+0x93>;
0x511d0a;:0;ADD	$0x20,%RSP;
0x511d0e;:0;POP	%RBP;
0x511d0f;:0;RET;
0x511d10;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1761;MOV	%RSI,%RDI;
0x511d13;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1761;MOV	%RSI,0x8(%RSP);
0x511d18;:0;CALL	504e50 <PyObject_Hash>;
0x511d1d;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1761;MOV	%RAX,%RDX;
0x511d20;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1762;CMP	$-0x1,%RAX;
0x511d24;:0;JE	4275f6 <PyDict_GetItemWithError.cold>;
0x511d2a;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1767;MOV	0x8(%RSP),%RSI;
0x511d2f;:0;LEA	0x10(%RSP),%RCX;
0x511d34;:0;MOV	%RBP,%RDI;
0x511d37;:0;CALL	505980 <_Py_dict_lookup>;
0x511d3c;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1769;MOV	0x10(%RSP),%RAX;
0x511d41;:0;JMP	511cfa <PyDict_GetItemWithError+0x4a>;
0x511d43;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1770;CALLQ	0x34aa2f(%RIP);
0x511d49;/usr/local/src/conda/python-3.12.0/Objects/dictobject.c:1770;NOPL	(%RAX);
